source/css/_settings.themes.scss

```diff
@@ -431,8 +431,18 @@ $themes: (
       background-color: $darker;
     }
 
-    .c-header .c-primary-nav__link {
-      color: $darker;
+    .c-header {
+      .c-primary-nav__link {
+        color: $darker;
+        &.this-is-active, &:active{
```

Comment:
Formatting: These should be on two separate lines and please add a space after `&:active`. Should look like this - https://github.com/adventistchurch/alps/blob/3fbf1abffbb0ed9eb19a34bdab33dce454855118/source/css/_settings.themes.scss#L251-L252